thrift-gen-fix.awk

```diff
@@ -9,6 +9,11 @@ BEGIN {six=0; xrange=0}
 }
 
 {
-        gsub(/from ttype/, "from .ttype",$0); 
-        print
+    gsub(/from ttype/, "from .ttype", $0);
+    gsub( \
+        /self.__dict__.iteritems\(\)/, \
+        "getattr(self.__dict__, 'iteritems', self.__dict__.items)()", \
```

Comment:
looks a bit complicated, I would rather use the helper function from `six` module, just like we do in the main code.

Reply:
Then maybe the following could be better.

```awk
BEGIN {six=0}

/^$/ {
    if (six == 0) {
        print "from six import iteritems";
        print "from six.moves import xrange";
    }
    six = 1
}

{
    gsub(/from ttype/, "from .ttype", $0);
    gsub(/self.__dict__.iteritems\(\)/, "iteritems(self.__dict__.items)", $0);
    print
}
```